app/assets/javascripts/angular/controllers/series/vote.js.erb

```diff
@@ -0,0 +1,274 @@
+app.controller('SeriesVoteCtrl', ['$scope', '$http', '$location', '$routeParams', function($scope, $http, $location, $routeParams){
+
+  $('.showResult').hide();
```

Comment:
Another interesting way of doing this in the future is by using `data binding` and `ng-show`:
```js
$scope.showsRecom = [false, false, false, false]
$scope.noRecom = false
```

And then in the view:
```html
<div class="showRecom1" ng-show="showsRecom[1]">
   ....
</div>
```